articles/anomaly-detection/index.md

```diff
@@ -16,82 +16,99 @@ useCase: customize-anomaly-detection
+
+### Trigger: 10 failed login attempts
 
-**Actions**:
-* Send an email to the affected user (The email can be [customized](#customize-the-blocked-account-email))
-* Block the suspicious IP address
+This trigger occurs when there are 10 failed login attempts into a single account from the same IP address.
 
-::: panel Note
-The way this anomaly protection works is that if user with "user_id1" signs in from IP1 and fails to login consecutively for 10 attempts their login from this IP - IP1 will be blocked. Another user, say "user_id2" signing in from the same IP (IP1) will not be blocked. The mechanism to clear this block is described below.
+#### Actions
 
-Currently the default trigger amount of 10 cannot be changed.
+* Send an email to the affected user. (The email can be [customized](#customize-the-blocked-account-email).)
+* Block the suspicious IP address for that user.
+
+::: note
+The default trigger amount of 10 cannot be changed.
 :::
 
+#### Clear block
+
 If this block is triggered, it can be cleared the following ways:
 
-* An administrator removes the block via the [Dashboard](${manage_url}) (by clicking **unblock for all IPs** under the **ACTIONS** button when viewing the user's details) or by using the [Management API](/api/management/v2#!/User_Blocks/delete_user_blocks) ;
-* The User clicks on the "unblock" link provided in the email sent when the block went into effect;
-* The User changes their password.
+* An administrator removes the block via the [Dashboard](${manage_url}) (by clicking **unblock for all IPs** under the **ACTIONS** button when viewing the user's details) or by using the [Management API](/api/management/v2#!/User_Blocks/delete_user_blocks).
+* The user clicks on the **unblock** link provided in the email sent when the block went into effect.
+* The user changes their password.
 
-**Trigger:** *100* failed login attempts from a single IP address using different usernames, all with incorrect passwords in 24 hours. Or *50* sign ups attempts per minute from the same IP address.
+### Triggers: 100 failed login attempts *or* 50 sign up attempts
 
-**Actions:**
-* Notify dashboard administrator(s)
-* Block suspicious addresses
+A trigger occurs when there are 100 failed login attempts from one IP address using different usernames with incorrect passwords in 24 hours. 
 
-If this block is triggered, additional access attempts are released one at a time over the course of 24 hours until 100 attempts are allocated. More specifically, you will gain 100 attempts / 24 hours * 60 minutes = approximately 1 additional attempt every 15 minutes.
+Another trigger occurs if there are 50 sign up attempts per minute from the same IP address.
 
-Auth0 does email the dashboard administrator(s) when this block is triggered. Within this email there's a link the owner can click on to remove the block.
+#### Actions
 
-#### Enable or Disable Brute Force Protection
+* Notify dashboard administrator(s).
+* Block suspicious addresses.
```

Comment:
Maybe explicitly say block suspicious address for 15 minutes? 🤔

Reply:
Changed